examples/cjs/cjs.py

```diff
@@ -57,24 +65,184 @@ def model(data):
                                 infer={"enumerate": "parallel"})
                 mu_y_t = rho * z
                 pyro.sample("y_{}".format(t), dist.Bernoulli(mu_y_t),
-                            obs=data[:, t])
-            first_capture_mask |= data[:, t].byte()
+                            obs=capture_history[:, t])
+            first_capture_mask |= capture_history[:, t].byte()
+
+
+"""
+In our second model variant there is a survival probability phi_t for 6
+of the 7 years of the capture data; each phi_t is treated as a fixed effect.
+"""
+
+
+def model_2(capture_history, sex):
+    N, T = capture_history.shape
+    rho = pyro.sample("rho", dist.Uniform(0.0, 1.0))  # recapture probability
+
+    z = torch.ones(N)
+    first_capture_mask = torch.zeros(N).byte()
+    # we create the plate once, outside of the loop over t
+    dippers_plate = pyro.plate("dippers", N, dim=-1)
```

Comment:
What is a dipper?

Reply:
see top of file: it's norway's national bird